### PR TITLE
build: bump version and CHANGELOG for release

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Prepare release. Bump version and doc changelog.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+| ------------------------ | ------- | ------------------- |
+| `@dfinity/ckbtc`         | v2.1.0  | Enhanced 🔧         |
+| `@dfinity/cmc`           | v2.1.0  | Enhanced 🔧         |
+| `@dfinity/ic-management` | v2.1.0  | Enhanced 🔧         |
+| `@dfinity/ledger-icp`    | v2.1.0  | Enhanced 🔧         |
+| `@dfinity/ledger-icrc`   | v2.1.0  | Enhanced 🔧         |
+| `@dfinity/nns`           | v3.1.0  | Enhanced 🔧         |
+| `@dfinity/nns-proto`     | v1.0.0  | Unchanged           |
+| `@dfinity/sns`           | v2.1.0  | Enhanced 🔧         |
+| `@dfinity/utils`         | v1.1.0  | Breaking Changes ⚠️ |
```
Comment (lmuntaner) [inline CHANGELOG.md:17]: Shouldn't this be `v2.0.0`?

**Review by lmuntaner — COMMENTED** [review]
One smal change

File: CHANGELOG.md (hunk 1 above)
Comment (peterpeterparker) [reply CHANGELOG.md]: Yes thx

**Review by lmuntaner — APPROVED** [review]
Thanks!

**Pull request merged**